NakedJavaScript/mensa-projekt
# M37   fix contactform

# What has changed?

Added working Contact-form and changed some other stuff, look at the changes

# How to test? 

Go to "Kontakt", fill the contact form, send an mail, check out the inbox https://inboxbear.com/q/864xjrw/2ffswdg if you receive a mail

Try to break it or find some shit that's ugly or smth
